internal/service/skrclient/client_proxy.go

```diff
@@ -156,6 +158,12 @@ func (p *ProxyClient) List(ctx context.Context, obj client.ObjectList, opts ...c
 	return nil
 }
 
+// Apply implements client.Apply
+// Will be implemented in https://github.com/kyma-project/lifecycle-manager/issues/2707
+func (p *ProxyClient) Apply(ctx context.Context, applyConfig machineryruntime.ApplyConfiguration, opts ...client.ApplyOption) error {
+	return ErrNotImplemented
```

Comment:
It's probably fine to do a dummy implementation here that returns an error, because until now the `Apply` method did not exists yet and is not used. Just make it clear here to `_` parameter variables. And let first sentence of the method's comment reflect that this will only return and error and is a dummy implementation.